src/components/plots/areaPlot.ts

```diff
@@ -77,6 +80,59 @@ export module Plots {
       });
     }
 
+    private _generateLineDrawSteps() {
+      var drawSteps: Drawers.DrawStep[] = [];
+      if (this._dataChanged && this._animate) {
+        var attrToProjector = this._generateLineAttrToProjector();
+        var xProjector = Plot._scaledAccessor(this.x());
+        var yProjector = Plot._scaledAccessor(this.y());
+
+        var definedProjector = (d: any, i: number, dataset: Dataset) => {
+          var positionX = xProjector(d, i, dataset);
+          var positionY = yProjector(d, i, dataset);
+          return positionX != null && positionX === positionX &&
+                 positionY != null && positionY === positionY;
+        };
+
+        attrToProjector["d"] = (datum: any, index: number, dataset: Dataset) => {
+          return d3.svg.line()
+                       .x((innerDatum, innerIndex) => xProjector(innerDatum, innerIndex, dataset))
+                       .y((innerDatum, innerIndex) => this._getResetYFunction()(innerDatum, innerIndex, dataset))
+                       .defined((innerDatum, innerIndex) => definedProjector(innerDatum, innerIndex, dataset))(datum, index);
+        };
+        drawSteps.push({attrToProjector: attrToProjector, animator: this._getAnimator("reset")});
+      }
+      drawSteps.push({attrToProjector: this._generateLineAttrToProjector(), animator: this._getAnimator("main")});
+      return drawSteps;
+    }
+
+    private _generateLineAttrToProjector() {
+      var lineAttrToProjector = this._generateAttrToProjector();
+      var fillProjector = lineAttrToProjector["fill"];
+      lineAttrToProjector["stroke"] = fillProjector;
+      lineAttrToProjector["fill"] = () => "none";
```

Comment:
The area `<path>` should never have a stroke, so setting it as a CSS style overrides any `attr` choices.